portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/SPA.testcase

```diff
@@ -35,4 +35,59 @@ definition {
 		AssertVisible.assertVisibleNoSPAWait(locator1 = "//html[contains(@class,'lfr-spa-loading')]");
 	}
 
+	@description = "LRQA-29748: Set the user notification timeout to 1 second"
+	@priority = 4
+	test SPANotificationSetTimeout {
+		property osgi.module.configurations = "userNotificationTimeout=1000";
```

Comment:
Do we need this property if we're already modifying it on the UI in System Settings?